extensions/notification/destination/KafkaNotificationDestination.js

```diff
@@ -1,5 +1,36 @@
 const NotificationDestination = require('./NotificationDestination');
 const KafkaProducer = require('./KafkaProducer');
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const notificationSize = ZenkoMetrics.createCounter({
+    name: 'notification_queue_processor_notification_size',
+    help: 'Total size of the notifications that were sent',
+    labelNames: ['target'],
+});
+
+const deliveryLag = ZenkoMetrics.createHistogram({
+    name: 'notification_queue_processor_notification_delivery_delay_sec',
+    help: 'Difference between the time a notification is sent and when it gets delivered',
+    labelNames: ['target', 'status'],
+    buckets: [0.0000001, 0.000001, 0.00001, 0.0001, 0.001, 0.01, 1, 10, 100, 1000, 10000],
```

Comment:
there is probably some cleanup to be done in this list of buckets : i doubt we would have GET operations take 10'000 seconds or 0.1 nano-second...